source/core/ut_output_buffer.pkb

```diff
@@ -0,0 +1,64 @@
+create or replace package body ut_output_buffer as
+
+  type tt_outputs is table of clob index by varchar2(128);
+
+  g_outputs tt_outputs;
+
+  procedure add_line_to_buffer(a_output_id varchar2, a_text varchar2) is
+  begin
+    if a_text is not null then
+      add_to_buffer(a_output_id, a_text||chr(10));
+    end if;
+  end;
+
+  procedure add_to_buffer(a_output_id varchar2, a_text clob) is
+    l_clob clob;
+  begin
+    if a_text is not null then
```

Comment:
and `dbms_lob.getlength(a_text)>0` as after `dbms_lob.createtemporary` clob variable is not null though the length is 0.